### PR TITLE
Prepare release 0.4.1

### PR DESCRIPTION
**Pull request by pablooliveira**
@yohanchatelain could you please check that I did not forget anything significant in the CHANGELOG ?

### CODE REVIEW

File: CHANGELOG [hunk 1]
```diff
@@ -1,5 +1,22 @@
 Verificarlo CHANGELOG
 
+# [v0.4.1] 2020/10/22
+
+## Added
+  * Add support for function instrumentation. Through a new compiler pass, backends can operate at the function granularity. Function instrumentation provides call site and call stack information and allows to instrument the scalar inputs and output of functions.
+  * VPREC backends takes advantage of function instrumentation to enable per function and per (scalar) argument exploration of variable precision
```
Comment (yohanchatelain) [inline CHANGELOG:7]: VPREC backend

**Review by yohanchatelain — COMMENTED** [review]
Just one small typo but otherwise LGTM :+1:, thanks for the quick push

**Review by yohanchatelain — CHANGES_REQUESTED** [review]
Just one small typo but otherwise LGTM +1, thanks for the quick push

File: CHANGELOG (hunk 1 above)
Comment (pablooliveira) [reply CHANGELOG]: Good catch !

**Review by yohanchatelain — APPROVED** [review]

**Pull request merged**